dev-libs/libp11/libp11-0.4.3.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="A PKCS#11 wrapper library"
+DESCRIPTION="libp11 is a library implementing a thin layer on top of PKCS#11 \
+API to make using PKCS11 implementations easier"
+HOMEPAGE="https://github.com/opensc/libp11"
+COPYRIGHT="2016 Michal Trojnara
+		2015-2016 Nikos Mavrogiannopoulos
+		2007-2009 Andreas Jellinghaus
+		2011 Martin Paljak"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/releases/download/libp11-$portVersion/libp11-$portVersion.tar.gz"
+CHECKSUM_SHA256="938bf73fc57ef8b543c6a2ceab617d7d83ccf60456241b73a61adb5ca37cd7c5"
+SOURCE_DIR="libp11-$portVersion"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libp11$secondaryArchSuffix = 2.4.3 compat >= 2
+	lib:libp11$secondaryArchSuffix
```

Comment:
Please replace
```
	libp11$secondaryArchSuffix = 2.4.3 compat >= 2
	lib:libp11$secondaryArchSuffix
```
by
```
	libp11$secondaryArchSuffix = $portVersion
	lib:libp11$secondaryArchSuffix = 2.4.3 compat >= 2
```
(libp11 is the name of the package, while lib:libp11 is the lib itself. The first needs a $portVersion, while the  second needs the "2.4.3".)
BTW, I'm pretty sure is what will fix the error you are getting:
```
No such file or directory: '/boot/home/haikuports/dev-libs/libp11/work-0.4.3'
```